cardigan_beta/.storybook/preview-head.html

```diff
@@ -1,3 +1,18 @@
 <link rel="stylesheet" href="css/styleguide.css" />
 <link rel="stylesheet" href="css/non-critical.css" />
 <script src="js/app.js" id='app-js'></script>
+
+
+<link rel="shortcut icon" href="{{ path(frctl.theme.get('favicon')) }}" type="image/ico">
```

Comment:
How comes we have Nunjucks/Fractal stuff in Storybook?

Reply:
Because I am a professional copy / paster.